swim/memberlist_test.go

```diff
@@ -201,9 +201,12 @@ func (s *MemberlistTestSuite) TestMultipleUpdates() {
 }
 
 func (s *MemberlistTestSuite) TestUpdateTriggersReincarnation() {
+	source := "192.0.2.1:1234"
+	s.NotEqual(source, s.m.local.Address, "this test relies on the source and the target of the change to be different")
```

Comment:
Isn't is easier to use an "impossible" source to prevent flaky tests? (like: `355.1.1.1:1234` or `192.0.2.300:1234`)